drivers/gpio/Kconfig

```
@@ -96,6 +96,7 @@ config GPIO_ENABLE_DISABLE_INTERRUPT
 
 # zephyr-keep-sorted-start
 source "drivers/gpio/Kconfig.ad559x"
+source "drivers/gpio/Kconfig.adi_max22017"
```

For consistency please rename to:
```suggestion
source "drivers/gpio/Kconfig.max22017"
```